product/charts/miq_reports/vim_perf_hourly_middleware_datasource.yaml

```diff
@@ -39,15 +40,17 @@ col_order:
 - mw_ds_timed_out
 - mw_ds_average_get_time
 - mw_ds_average_creation_time
+- mw_ds_max_wait_time
 
 # Column titles, in order
 headers:
 - Date/Time
 - Available
 - In Use
 - Time-out
-- Wait Time (avg)
+- Get Time (avg)

```

Comment:
The previous code fetche a metric named "Datasource Pool Metrics~Average Get Time" and show it as "Wait Time (avg)" , I renamed it to "Get Time (avg)", but after reading the definition of the GetTime metric is not clear to me what is the difference.

**AverageGetTime**: The average time spent obtaining a physical connection.
**MaxGetTime**: The maximum time for obtaining a physical connection.
**MaxWaitTime**: The maximum wait time for a connection.

Also, I saw in the documentation and there is no metric named **AverageWaitTime**.